### DISCUSSION TITLE
RT from stimulus onset but show stimulus very short

### DISCUSSION THREAD

**Discussion by NadineDijkstra** [jspsych/jsPsych#1109] [Q&A] [answered]
Hi hi, I want to show a stimulus quite briefly (200 ms) and then log a response from stimulus onset which can happen after stimulus offset. I thought that I could use the image-keyboard-response plugin for that by setting the 'stimulus_duration' parameter to my stimulus duration and the 'trial_duration' to be longer than that, like so: 

![image](https://user-images.githubusercontent.com/18414967/95459483-a5a2e980-096b-11eb-8699-9dfbc85a2585.png)

However, this gives me the following error: 

![image](https://user-images.githubusercontent.com/18414967/95459552-bb181380-096b-11eb-8002-7379d1ca50a6.png)

Could you tell me what I am doing wrong? Thank you!

**Comment by wenbo-lin** [comment, marked as answer]
Switch = with : 
`stimulus_duration: 200`

**Comment by NadineDijkstra** [comment]
Omg, I must be so tired, thank you!
